lib/rules/one-var.js

```diff
@@ -28,6 +49,7 @@ module.exports = function(context) {
      */
     function startFunction() {
         functionStack.push(false);
+        startBlock();
```

Comment:
Jeebus. Thanks.

I must have a mental block on this PR. I keep doing the stupid.